src/video/emscripten/SDL_emscriptenopengles.c

```diff
@@ -101,7 +101,7 @@ SDL_GLContext Emscripten_GLES_CreateContext(SDL_VideoDevice *_this, SDL_Window *
 
     context = emscripten_webgl_create_context(window_data->canvas_id, &attribs);
 
-    if (context < 0) {
+    if (context == 0) {
```

Comment:
```suggestion
    if (!context) {
```
This more clearly indicates that it's an alias to context handle/pointer.